velox/docs/functions/presto/math.rst

```diff
@@ -6,6 +6,12 @@ Mathematical Functions
 
     Returns the absolute value of ``x``.
 
+.. function:: beta_cdf(a, b, value) -> double
```

Comment:
Let's move this function to a new section "Probability Functions: cdf" like in https://prestodb.io/docs/current/functions/math.html

Let's also convert "Beta cdf" into a link probably to https://en.wikipedia.org/wiki/Beta_distribution

Reply:
moved beta_cdf into new section